crates/levm/Makefile

```diff
@@ -0,0 +1,28 @@
+.PHONY: all test clippy fmt usage lint eth-tests
+
+all: test clippy fmt
+
+usage:
```

Comment:
you can use something like https://github.com/lambdaclass/ethereum_rust/blob/main/Makefile#L4